lib/livebook/runtime/evaluator.ex

```diff
@@ -620,6 +620,73 @@ defmodule Livebook.Runtime.Evaluator do
     end
   end
 
+  defp eval("erlang", code, binding, env) do
+    try do
+      erl_binding =
+        binding
+        |> List.foldl(:erl_eval.new_bindings(), fn {name, value}, erl_binding ->
```

Comment:
Erlang allow map bindings, they should be more performant, let's also use reduce.

```suggestion
        |> Enum.reduce(%{}, fn {name, value}, erl_binding ->
```

Reply:
Hah, good point. I took `ierl` as a reference (which is GPL but completely written by me), so I didn't bother checking for updates here :)